src/config/feeds-schema.ts

```diff
@@ -205,5 +206,24 @@ export default {
       required: ['params'],
       additionalProperties: false,
     },
+    OnChainDataFetcher: {
+      properties: {
+        name: { const: 'OnChainData' },
+        params: {
+          type: 'object',
+          properties: {
+            address: { type: 'string' },
```

Comment:
do you think we can use contract name through ENS?

Reply:
yes ENS can be used but this is not good idea. we should use addresses always.